services/cloudfront/src/main/java/software/amazon/awssdk/services/cloudfront/model/CustomSignerRequest.java

```diff
@@ -50,6 +50,7 @@ private CustomSignerRequest(DefaultBuilder builder) {
         this.expirationDate = builder.expirationDate;
```

Comment:
I know it was already there, can we remove the null validation for `expirationDate` from `SigningUtils.buildCustomPolicyForSignedUrl()` and do a `Validate.notNull()` here? This way it will cover `cookiesForCustomPolicy()` as well.